### DISCUSSION TITLE
Running KSP2 mods (BepInEx) on Linux via Proton

### DISCUSSION THREAD

**Discussion by PacifickFR42** [KSP-CKAN/CKAN#3982] [Q&A] [answered]
Hello world, 

I try to use CKAN on Linux Ubuntu 22.04 with KSP 2
But no mods appear in the game, nor even the "Mods" menu in the main menu of KSP 2
I tried the Windows version with mono, same result.

Do you have any idea of the problem?
THANKS.

**Comment by HebaruSan** [comment]
Did you _install_ any mods with CKAN?

**Reply by PacifickFR42** [reply]
Yes, Bepinex, K2-D2, SpaceWrap, UITK.

**Reply by HebaruSan** [reply]
If you open the game folder, is there a `BepInEx/plugins` folder there?

**Reply by PacifickFR42** [reply]
Yes !

**Reply by PacifickFR42** [reply]
![Capture d’écran du 2023-12-30 19-44-12](https://github.com/KSP-CKAN/CKAN/assets/100038348/e7a48157-5195-4c42-9b69-dab8066d4739)

**Reply by HebaruSan** [reply]
OK, that means that the CKAN part of this worked fine.

How are you running a Windows game on Linux? What tools are you using?

**Reply by PacifickFR42** [reply]
SteamLauncher with Proton
![Capture d’écran du 2023-12-30 19-46-38](https://github.com/KSP-CKAN/CKAN/assets/100038348/3fd064bf-2a1c-48e4-899e-4f2fa11f3cc1)

**Reply by HebaruSan** [reply, marked as answer]
This is a BepInEx question rather than a CKAN question. Here are that team's instructions for Proton:

https://docs.bepinex.dev/articles/advanced/proton_wine.html

Good luck! 👍

**Reply by PacifickFR42** [reply]
I'll look at that, thanks!

**Reply by PacifickFR42** [reply]
Thank you very much my friend, it works perfectly!!
